journal/modernized/06-chapter-3.adoc

```diff
@@ -463,7 +463,7 @@ that on the arrival of the British army at Bremen,
 the kind and cordial treatment they received from its inhabitants,
 was strongly contrasted with the hard-heartedness and hostility of the Dutch.
 'It was something like a dream,' says a witness and partaker of their
-sufferings:'--'we who had lately been so buffeted about,
+sufferings."We who had lately been so buffeted about,
```

Comment:
I just talked to Jared for a while on the phone about quotes and single quotes. I'll slack you some details today. This chunk right here in Daniel Wheeler is a footnote, where there are a bunch of quotations put into single (straight) quotes. I think all the single quotes in this footnote can be changed to regular quotes, but you'll need to use the little backtick thing that is in the style guide.  So for example it would be .... sufferings. "`We who had lately....